components/clientComponents/forms/FormattedDate/FormattedDate.tsx

```diff
@@ -121,7 +121,7 @@ export const FormattedDate = (props: FormattedDateProps): React.ReactElement =>
       {meta.error && <ErrorMessage id={"errorMessage" + id}>{meta.error}</ErrorMessage>}
 
       <div className="inline-flex gap-2">
-        <input type="hidden" {...field} />
+        <input type="hidden" {...field} value={field.value || ""} />
```

Comment:
Noting this is a fix for uncontrolled / controlled component --- not related to this PR but fixes.